packages/calcite-components/src/components/tree-item/tree-item.scss

```diff
@@ -57,14 +61,16 @@
     @apply min-h-[2.75rem];
     .node-container {
       .checkbox,
-      .chevron,
       .checkmark,
       .bullet-point {
         margin-inline: theme("margin.3");
       }
       .icon-start {
         margin-inline: theme("margin.3");
       }
+      .chevron {
+        padding: 0.7rem;
```

Comment:
Should this be using a `theme` or a spacing prop/token? cc @ashetland @driskull

Reply:
Good question. cc @ashetland 👍